chart/prometheus-msteams/README.md

```diff
@@ -86,6 +87,40 @@ helm upgrade --install prometheus-msteams \
 
 Otherwise you can also set the value by specifying the template data directly via values file.
 
+### Customise Messages per MS Teams Channel
+
+This application uses a [Default Teams Message Card Template](./prometheus-msteams/card.tmpl) to convert incoming Prometheus alerts to teams message cards.
+To define a custom message template per MS Teams channel you can use the following configuration.
+
+```yaml
+connectorsWithCustomTemplates:
+  - request_path: /alert2
+    webhook_url: <webhook>
+    escape_underscores: true
+```
+
+Simply create a new file that you want to use as your custom template (for example, `custom-card.tmpl`).
+You can use the `--set-file` flag to set the value from this file:
+
+```bash
+helm upgrade --install prometheus-msteams \
+  --namespace default -f config.yaml \
+  --set-file "connectorsWithCustomTemplates[0].template_file=custom-card.tmpl" \
+  prometheus-msteams/prometheus-msteams
+```
+
+Otherwise you can also set the value by specifying the template data directly via values file:
+
+```yaml
+connectorsWithCustomTemplates:
+  - request_path: /alert2
+    webhook_url: <webhook>
+    escape_underscores: true
+    template_file: |
+      {{ define "teams.card" }}
+      {...}
+      {{ end }}
+```
 
 ### Helm Configuration
```

Comment:
This table still displays `connectors_with_custom_templates` instead of `connectorsWithCustomTemplates`. I was thinking: does this naming change introduce a major change, although it is actually a fix as it didn't work before? What do you think?

Reply:
I guess you are right. You can not use the old name anymore, so it should be a major change. Shall we update the version to `1.0.0` then?

Reply:
Yes let's do it. I can't directly link the line in the readme, this is what I meant:
![image](https://user-images.githubusercontent.com/18685317/100153668-b7e9d000-2ea4-11eb-8756-c3b7b233cabe.png)

See that `connectors_with_custom_templates`?

Reply:
Ahh right, changed it and updated the chart version.